examples/baselines/act/README.md

```diff
@@ -0,0 +1,96 @@
+# Action Chunking with Transformers (ACT)
+
+Code for running the ACT algorithm based on ["Learning Fine-Grained Bimanual Manipulation with Low-Cost Hardware"](https://arxiv.org/pdf/2304.13705). It is adapted from the [original code](https://github.com/tonyzhaozh/act).
+
+## Installation
+
+To get started, we recommend using conda/mamba to create a new environment and install the dependencies
+
+```bash
+conda create -n act-ms python=3.9
+conda activate act-ms
```

Comment:
so the conda env act-ms is created and you do a local pip install. However a simple setup.py file is still missing, can you create that?

Reply:
I created a simple setup.py.